openfisca_uk/parameters/tax/national_insurance/class_1/rates/employee/additional.yaml

```
@@ -1,14 +1,8 @@
-description: Additional contribution rate by employees (on earnings above the UEL)
+description: The Class 1 NI additional rate is paid on employment earnings above the Upper Earnings Limit
```

```suggestion
description: The Class 1 National Insurance additional rate is paid on employment earnings above the Upper Earnings Limit
```
And below - seems that others avoid acronyms in descriptions